docs/source/reference/label_format/Keypoints2D.rst

```diff
@@ -112,6 +112,80 @@ Besides :ref:`reference/label_format/CommonSubcatalogProperties:attributes infor
 it also has :attr:`~tensorbay.label.label_keypoints.Keypoints2DSubcatalog.keypoints`
 to describe a set of keypoints corresponding to certain categories.
 
+The catalog with only Keypoints2D subcatalog is typically stored in a json file as follows::
+
+    {
+        "KEYPOINTS2D": {                                  <object>*
+            "description":                                <string>! -- Subcatalog description, (default: "").
+            "isTracking":                                <boolean>! -- Whether this type of label in the dataset contains tracking
+                                                                       information, (default: false).
+            "keypoints": [
+                {
+                    "number":                            <integer>* -- The number of key points.
+                    "name":                                <array>  -- The name of each key point that corresponds to the
+                                                                       "keypoints2d" list in the Keypoints2D label via index.
+                    "skeleton": [                          <array>  -- Key point skeleton for visualization.
+                        [<index>, <index>],                <array>  -- Each array represents a line segment. The skeleton is formed
+                                                                       by connecting these lines corresponding to index.
```

Comment:
`each array` or `each element`?
Is `corresponding to index` reasonable?

Reply:
Here, the element is <index>.

Reply:
The skeleton is formed  by connecting these lines corresponding to the value of <index>.